commcare_connect/templates/prelogin/includes/hubspot_embed.html

```
@@ -0,0 +1,5 @@
+{% if HUBSPOT_API_ID %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)